app/components/UserList/UserList.tsx

```diff
@@ -1,12 +1,52 @@
 import _ from 'lodash';
-import React, { useMemo } from 'react';
-import { useSelector } from 'react-redux';
+import React, { createRef, useMemo } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import { appConfigSelector } from '../../modules/appConfig';
 import UserListItem from './UserListItem';
+import { conferenceSelector, setMiniRoom } from '../../modules/conference';
+import classes from './UserList.css';
+
+// eslint-disable-next-line react/prop-types
+const ConferenceSwitchButton = ({ miniRoom, currentMiniRoom }) => {
+  const dispatch = useDispatch();
+
+  const buttonRef = createRef<HTMLButtonElement>();
+
+  function handleClick() {
+    dispatch(setMiniRoom(miniRoom));
+    if (buttonRef.current) buttonRef.current.blur();
+  }
+
+  let buttonClasses = 'btn-dark';
+  if (miniRoom === currentMiniRoom) {
+    buttonClasses = 'btn-success active';
+  }
```

Comment:
```suggestion
  const buttonClasses = useMemo(() => {
    if (miniRoom === currentMiniRoom) {
        return 'btn-success active';
    }
    
    return 'btn-dark'
  }, [miniRoom, currentMiniRoom])
  
```

or maybe some lib like classnames would come handy for these cases